src/rust/engine/src/cffi_build.rs

```diff
@@ -173,6 +173,15 @@ fn main() -> Result<(), CffiBuildError> {
 
   config.compile("libnative_engine_ffi.a");
 
+  if cfg!(target_os = "macos") {
+    // N.B. On OSX, we force weak linking by passing the param `-undefined dynamic_lookup` to
+    // the underlying linker. This avoids "missing symbol" errors for Python symbols
+    // (e.g. `_PyImport_ImportModule`) at build time when bundling the CFFI C sources.
+    // The missing symbols will instead by dynamically resolved in the address space of the parent
+    // binary (e.g. `python`) at runtime - obviating a need to link to libpython.
```

Comment:
By obviating, you mean "removing", right? Consider a more accessible synonym like "removing".

Reply:
This was just copied from a now-deleted file. But I agree - reworded :)